smoke-test/src/test/resources/device-config/SmokeTests.xml

```diff
@@ -0,0 +1,7 @@
+<foreign-source xmlns="http://xmlns.opennms.org/xsd/config/foreign-source" name="SmokeTests" date-stamp="2022-10-04T13:38:18.589Z">
+   <scan-interval>1d</scan-interval>
+   <detectors>
+      <detector name="SSH" class="org.opennms.netmgt.provision.detector.ssh.SshDetector"/>
```

Comment:
We do not need this detector. Even if it is responding quite fast, we can save the cycles by not letting it run for every node.

Reply:
Removed as did not help in speeding up the process